sycl/include/CL/sycl/backend/level_zero.hpp

```diff
@@ -0,0 +1,81 @@
+//==--------- level_zero.hpp - SYCL Level-Zero backend ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+//#include <ze_api.h> This should be included from user code
```

Comment:
This should be aligned with `CL/sycl/backend/opencl.hpp`. It includes (though, indirectly) the `CL/cl.h` file.

Reply:
Fixed.